x/slashing/keeper/infractions.go

```diff
@@ -43,7 +43,7 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr crypto.Address, p
 	}
 
 	validator, err := k.sk.GetValidatorByConsAddr(ctx, consAddr)
-	if err == nil && !validator.IsInactivated() {
+	if err == nil && !validator.IsInactivated() && !validator.IsPaused() { // TODO: check other statuses Active | Inactive | Paused | Jailed
 		k.sk.HandleValidatorSignature(ctx, validator.ValKey, missed, signInfo.Mischance)
 	}
```

Comment:
When status move from Active to Inactive or Jailed, should all the counters be zero.